test/test_linalg.py

```diff
@@ -4680,7 +4704,7 @@ def test(n=10,                       # how many tests to generate
                 args.append(out_sublist)
                 self._check_einsum(*args, np_args=(equation, *np_operands))
 
-        test(100)
+        test(1000)
```

Comment:
1000!? How long does this test take in CI now (on CPU and CUDA)?

Reply:
Around 2-3s